eng/Version.Details.xml

```diff
@@ -3,39 +3,39 @@
   <ProductDependencies>
     <Dependency Name="Microsoft.WindowsDesktop.App.Ref" Version="5.0.0" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>5848b16d1fc803ea4892692b3b5dd9ee79955428</Sha>
+      <Sha>ec2af60c1eb8b2665745750f6445b3ff133178fb</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.WindowsDesktop.App" Version="5.0.1-servicing.20560.5" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="Microsoft.WindowsDesktop.App" Version="5.0.1-servicing.20566.1" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>31f95185b05655dda70e43968fe69a9361af8b2d</Sha>
+      <Sha>a8f8d26d9af8ae258d8687beaa4539dca6982962</Sha>
     </Dependency>
     <Dependency Name="Microsoft.WindowsDesktop.App.Runtime.win-x64" Version="5.0.1" CoherentParentDependency="Microsoft.NET.Sdk">
       <Uri>https://github.com/dotnet/windowsdesktop</Uri>
-      <Sha>31f95185b05655dda70e43968fe69a9361af8b2d</Sha>
+      <Sha>a8f8d26d9af8ae258d8687beaa4539dca6982962</Sha>
     </Dependency>
-    <Dependency Name="VS.Redist.Common.WindowsDesktop.TargetingPack.x64.5.0" Version="5.0.0-rtm.20529.4" CoherentParentDependency="Microsoft.NET.Sdk">
+    <Dependency Name="VS.Redist.Common.WindowsDesktop.TargetingPack.x64.5.0" Version="5.0.0-rtm.20520.1" CoherentParentDependency="Microsoft.NET.Sdk">
```

Comment:
@wli3 this is the downgrade it's complaining about (there may be others below, this is just the first one I noticed)

Reply:
@NikolaMilosavljevic @wtgodbe could you work together to see if this is expected? Seems like runtimes didn't do anything special

Reply:
It's correct. @mmitche did some work the other day to correct 5.0 dependencies to use the versions we actually shipped with RTM, rather than later 5.0.0 builds. This can be merged as-is